webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/db/Doobie.scala

```diff
@@ -172,12 +172,12 @@ object Doobie {
   )
 
   implicit val ReportRead: Read[Reports] = {
-    type R = (DateTime, RuleId, DirectiveId, NodeId, Int, String, String, DateTime, String, String)
+    type R = (DateTime, RuleId, DirectiveRId, NodeId, Int, String, String, DateTime, String, String)
```

Comment:
i didn't realize it would change the format on the reports from the agents

Reply:
it should be transparent (a different provided string to put where it should)